.github/workflows/alpine.yaml

```diff
@@ -8,21 +8,54 @@ on:
       - "alpine/**"
       - ".github/workflows/alpine.yaml"
       - "install_tooling.sh"
+  workflow_call:
+    secrets:
+      IMAGE_PREFIX:
+        description: "registry.com/loftsh in registry.com/loftsh/language:TAG"
+        required: true
 
 env:
-  IMAGE_PREFIX: loftsh
+  IMAGE_PREFIX: ${{ secrets.IMAGE_PREFIX }}
+  LANGUAGE: "alpine"
+  SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+
+defaults:
+  run:
+    working-directory: alpine
 
 jobs:
   alpine:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - tag: edge
+            target_tag: edge
+          - tag: latest
+            target_tag: latest
+          - tag: 3.14
+            target_tag: 3.14
+          - tag: 3.13
+            target_tag: 3.13
+          - tag: 3.12
+            target_tag: 3.12
```

Comment:
This looks very repetitive to me. Why not have `taget_tag` be the same as `tag` if `target_tag` is empty?

Reply:
will look at this later